library/src/main/java/com/otaliastudios/zoom/internal/Dispatcher.kt

```diff
@@ -0,0 +1,42 @@
+package com.otaliastudios.zoom.internal
+
+import com.otaliastudios.zoom.ZoomApi
+import com.otaliastudios.zoom.ZoomEngine
+import com.otaliastudios.zoom.ZoomEngine.Listener
+
+internal class Dispatcher(private val engine: ZoomEngine) {
```

Comment:
I feel the same about this class name. What is it dispatching to and from whom and why?
There should be at least class documentation but I would also add docs to *any* method no matter the visibility.